migrations/versions/201608181522_ccd9d0858ff_add_display_order_to_personlinks.py

```diff
@@ -0,0 +1,36 @@
+"""Add display order to PersonLinks
+
+Revision ID: ccd9d0858ff
+Revises: 8763fe20f41
+Create Date: 2016-08-17 15:22:57.540435
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'ccd9d0858ff'
+down_revision = '90e05643cf5'
```

Comment:
might also want to rename the file to the current date to keep it in the right order